circuits/cpp/src/aztec3/circuits/abis/c_bind.cpp

```diff
@@ -77,14 +77,6 @@ template <size_t TREE_HEIGHT> void rightfill_with_zeroleaves(std::vector<NT::fr>
 
 }  // namespace
 
-// Note: We don't have a simple way of calling the barretenberg c-bind.
-// Mimic bbmalloc behaviour.
```

Comment:
Laziness on my part initially. Easy now at least to provide a way to call the function while iterating circuits.